src/main/kotlin/apply/application/MissionService.kt

```diff
@@ -54,7 +74,8 @@ class MissionService(
     }
 
     fun getById(id: Long): MissionResponse {
-        return missionRepository.getById(id).let(::MissionResponse)
+        val mission = missionRepository.getById(id)
+        return MissionResponse(mission)
```

Comment:
c: 굳이 바꾸지 않아도 되는 코드였네요.

```suggestion
        return missionRepository.getById(id).let(::MissionResponse)
```